### PR TITLE
unused_var object field composition spec

### PR DESCRIPTION
**Pull request by nmccready**
Just wondering if this should not be failing? Is this already fixed in 1.X branch?

### CODE REVIEW

File: test/ScopeLinter/unused.coffee [hunk 1]
```diff
@@ -273,3 +273,16 @@ describe "ScopeLinter/unused", ->
         ), {
             unused_variables: true
         }).should.have.length(0)
+
+
+    it "matches composed string field objects", ->
```
Comment (nmccready) [inline test/ScopeLinter/unused.coffee:278]: This spec is failing. Should travis.yml also be added to show that this spec fails publicly?

**Comment by za-creature** [comment]
Nice catch!

Your test was slightly broken:

1. `#{field}` inside "" actually got interpolated within the test itself, and would fail with a `ReferenceError: field is not defined`
2. you'd _always_ get a unused error for "obj" as it's assigned to but never read

However, you are correct in that the linter didn't recognize the interpolated expression as a read for `field`! Fixed in 0.4.4

File: test/ScopeLinter/unused.coffee (hunk 1 above)
Comment (za-creature) [reply test/ScopeLinter/unused.coffee:278]: I actually use CircleCI (`circle.yml`) and I'm unsure if that can be configured to pick up remote PRs; it works for local branches, I think

**Comment by nmccready** [comment]
nice ty

**Pull request closed without merging**